.all-contributorsrc

```diff
@@ -520,6 +520,15 @@
       "contributions": [
         "content"
       ]
+    },
+    {
+      "login": "ajveltri22",
+      "name": "ajveltri22",
```

Comment:
Can I add your real name here, @ajveltri22? (the bot repeats a github username when it can't infer the real name)

```suggestion
      "name": "Anthony J Veltri",
```

Reply:
Yes, you can use my name! Thanks!